testing/plans/plan_templates/main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    leanspace = {
+      source = "leanspace/leanspace"
+    }
+  }
+}
+
```

Comment:
Can you add a test case with as all the input parameters ? To make sure that everything is parsed correctly